### PR TITLE
Add a file reading function in challenge utils.

### PR DESCRIPTION
**Pull request by RishabhJain2018**
@deshraj @taranjeet Please review the PR.

### CODE REVIEW

File: apps/challenges/utils.py [hunk 1]
```diff
@@ -6,3 +8,9 @@
 get_challenge_model = get_model_object(Challenge)
 
 get_challenge_phase_model = get_model_object(ChallengePhase)
+
+
+def get_file_content(file_path, mode):
```
Comment (taranjeet) [inline apps/challenges/utils.py]: May I know why are we adding this function? I am still not able to infer its appropriate use.

Comment (RishabhJain2018) [inline apps/challenges/utils.py]: Actually earlier we were reading all the data from the yaml file like `challenge_description` , `terms_and_conditions` etc. but as this content will be long, the yaml file configuration will not look good. So, now we are reading this content also from the different files, that's why I created this utility function to remove the redundancy in code. Please tell if I am not clear somewhere.

Comment (taranjeet) [inline apps/challenges/utils.py]: Okay I got it.

File: apps/challenges/utils.py [hunk 2]
```diff
@@ -6,3 +8,9 @@
...
+    if os.path.isfile(file_path):
+        with open(file_path, mode) as file_content:
```
Comment (taranjeet) [inline apps/challenges/utils.py]: The function intends to be general, thats great actually But here we are only reading the file and returning its content. If this is concerned, then I think mode should have a default value of read(`r`). Infact we can drop this mode as parameter if we are sure that there wont be any binary (mode=rb) file.

@deshraj : your thoughts here.

File: tests/unit/challenges/test_utils.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+from django.conf import settings
+
+from challenges.utils import get_file_content
+
+
+class BaseTestCase(unittest.TestCase):
+
+    def setUp(self):
+        self.test_file_path = os.path.join(settings.BASE_DIR, 'examples', 'example1', 'test_annotation.txt')
```
Comment (taranjeet) [inline tests/unit/challenges/test_utils.py]: Nice use of file. :)

File: apps/challenges/utils.py (hunk 2 above)
Comment (deshraj) [inline apps/challenges/utils.py]: I think this looks good for now and we can use `r` as default parameter later on.

**Pull request merged**